packages/utilities/src/BaseComponent.ts

```diff
@@ -241,31 +240,18 @@ function _makeSafe(obj: BaseComponent<{}, {}>, prototype: Object, methodName: st
     (obj as any)[methodName] = function (): any {
       let retVal;
 
-      try {
```

Comment:
I suppose this `try-catch` is the core meaning of this `_makeSafe` function. Now, after removing this `try-catch` logic, do we still need this `_makeSafe` function?

Reply:
Yes, though it might be more aptly named to something more clear. I was trying to minimize the change as much as possible.

Reply:
That sounds good to me!